config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+Kaminari.configure do |config|
+  config.default_per_page = 20
```

Comment:
Is it worth making these env vars?

Reply:
Not sure, in the document Matt was talking about having 20 results as a default and it is a value that can be overridden. What is the advantage of having this as an env variable?

Reply:
Yes you're right, there's no advantage as these are overridable defaults.